<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Similog</class>
 <widget class="QDialog" name="Similog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Similog computation setup</string>
  </property>
  <widget class="QDialogButtonBox" name="_dialog_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>391</width>
     <height>319</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="_table_label">
        <property name="text">
         <string>Table</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="_table">
        <property name="text">
         <string>albion.resistivity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="_pelt_sup_label">
        <property name="toolTip">
         <string>Maximum number of output markers</string>
        </property>
        <property name="text">
         <string>Number of markers</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="_nb_markers">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="_depth_min_label">
        <property name="toolTip">
         <string>Minimal depth from which data is considered, in a strict inequality scheme</string>
        </property>
        <property name="text">
         <string>Min depth</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="_depth_min">
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="_depth_max_label">
        <property name="toolTip">
         <string>Maximal depth from which data is considered, in a strict inequality scheme</string>
        </property>
        <property name="text">
         <string>Max depth</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="_depth_max">
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>200.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="_min_resistivity_label">
        <property name="toolTip">
         <string>Lower threshold for measures, in a strict inequality scheme</string>
        </property>
        <property name="text">
         <string>Min resistivity value</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="_value_min">
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="_max_resistivity_label">
        <property name="toolTip">
         <string>Upper threshold for measures, in a strict inequality scheme</string>
        </property>
        <property name="text">
         <string>Max resistivity value</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="_value_max">
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="_min_seg_label">
        <property name="toolTip">
         <string>Positive integer giving the minimum segment length (number of observations between changes)</string>
        </property>
        <property name="text">
         <string>Segmentation step</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="_min_seg">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="_log_normalize">
        <property name="text">
         <string>Log normalize</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="_transcribe_regressed_values">
        <property name="text">
         <string>Linear regression before transcription</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QCheckBox" name="_segmentize_with_pelt">
        <property name="text">
         <string>Segmentize with PELT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>396</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QProgressBar" name="_similog_progress">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="_compute_button">
      <property name="text">
       <string>Compute</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="_similog_info">
      <property name="text">
       <string>Click on 'Compute' button to run similog procedure</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_dialog_button</sender>
   <signal>accepted()</signal>
   <receiver>Similog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_dialog_button</sender>
   <signal>rejected()</signal>
   <receiver>Similog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_compute_button</sender>
   <signal>clicked()</signal>
   <receiver>Similog</receiver>
   <slot>_exec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
